plexapi/photo.py

```diff
@@ -158,6 +172,7 @@ def sync(self, resolution, client=None, clientId=None, limit=None, title=None):
         sync_item.contentType = self.listType
         sync_item.metadataType = self.METADATA_TYPE
         sync_item.machineIdentifier = self._server.machineIdentifier
+        sync_item.userRating = self.userRating
```

Comment:
Are you sure on this one? I don't remember a `userRating` field within SyncItem in the API.

Reply:
I am, tested and everything. This userRating value is stored in the metadata details table and not on the normal media table.

Reply:
[SyncItem object](https://github.com/pkkid/python-plexapi/blob/master/plexapi/sync.py) doesn't knows anything about `userRating` field, so the field wouldn't be processed in any way there. I'm not really getting it, why have you added it here?